From andrew@lunn.ch  Thu Feb 25 15:37:48 2010
From: Marek Lindner <lindner_marek@yahoo.de>
Date: Fri, 19 Feb 2010 16:18:07 +0100
Subject: Staging: batman-adv: remove obsolete variables
To: gregkh@suse.de
Cc: devel@linuxdriverproject.org, b.a.t.m.a.n@lists.open-mesh.net, Marek Lindner <lindner_marek@yahoo.de>, Andrew Lunn <andrew@lunn.ch>
Message-ID: <1266592691-13090-4-git-send-email-andrew@lunn.ch>


From: Marek Lindner <lindner_marek@yahoo.de>

This patch removes a variable that became obsolete since the skb handling
replaced the packet handling thread.

Signed-off-by: Marek Lindner <lindner_marek@yahoo.de>
Signed-off-by: Andrew Lunn <andrew@lunn.ch>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/batman-adv/routing.c |    2 --
 drivers/staging/batman-adv/routing.h |    1 -
 2 files changed, 3 deletions(-)

--- a/drivers/staging/batman-adv/routing.c
+++ b/drivers/staging/batman-adv/routing.c
@@ -35,8 +35,6 @@
 
 DECLARE_WAIT_QUEUE_HEAD(thread_wait);
 
-atomic_t exit_cond;
-
 void slide_own_bcast_window(struct batman_if *batman_if)
 {
 	HASHIT(hashit);
--- a/drivers/staging/batman-adv/routing.h
+++ b/drivers/staging/batman-adv/routing.h
@@ -22,7 +22,6 @@
 #include "types.h"
 
 extern wait_queue_head_t thread_wait;
-extern atomic_t exit_cond;
 
 void slide_own_bcast_window(struct batman_if *batman_if);
 void receive_bat_packet(struct ethhdr *ethhdr,
